components/UiKit/TypographyHomepage.tsx

```diff
@@ -13,6 +13,7 @@ export const Heading1 = styled.h1`
   @media ${device.S} {
     font-size: 42px;
   }
+  ${({ $colorBlack }) => $colorBlack && "color: #1F1F1F"};
```

Comment:
pls use `theme.blackLight` instead of hardcoded color.
And maby think about if black would be default color for Heading1 and $white would be optional as white is used only on the landign page.